extensions/wikia/AdEngine/js/template/bfaaDesktop.js

```diff
@@ -0,0 +1,110 @@
+/*global define, require*/
+define('ext.wikia.adEngine.template.bfaaDesktop', [
+	'ext.wikia.adEngine.adHelper',
+	'ext.wikia.adEngine.context.uapContext',
+	'ext.wikia.adEngine.provider.btfBlocker',
+	'ext.wikia.adEngine.slotTweaker',
+	'ext.wikia.adEngine.video.uapVideoAd',
+	'wikia.document',
+	'wikia.log',
+	'wikia.window',
+	require.optional('ext.wikia.aRecoveryEngine.recovery.tweaker')
+], function (
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has 9 parameters which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)